### ISSUE TITLE
Question about the attention block.

### ISSUE THREAD

**Issue by SongYupei** [DSaurus/DeepMultiCap#6]
Thanks for sharing the code of the paper. I have a question about the code in line 234-237.
https://github.com/DSaurus/DeepMultiCap/blob/4e93adfbdd60520d4edaa8bd8e461fe41d01c489/lib/model/DMCNet.py#L234 
``
geo_feature = torch.cat(geo_feature, dim=0)
print('attention input shape:{0}'.format(geo_feature.shape))
# [4, 1, 4, 320, 5000]        
if self.opt.coarse_part:
      geo_feature = self.attention(geo_feature, self.feature_fusion)
      # [1, 1, 4, 320, 5000]
      print('attention output shape:{0}'.format(geo_feature.shape))
``
the shape[0] of geo_feature is become from 4 to1, but geo_feature.shape[0] is level number, it is not the viewr number become one.
Any one can help me understand the probelem?
